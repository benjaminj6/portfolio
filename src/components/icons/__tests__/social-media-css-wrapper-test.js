/* eslint-env jest */
import { h } from 'preact'
import { shallow } from 'preact-render-spy'

import SocialMediaCssWrapper from '../social-media-css-wrapper'

test('Social Media CSS Wrapper snap', () => {
  const tree = shallow(
    <SocialMediaCssWrapper>Test Content</SocialMediaCssWrapper>
  )
  expect(tree).toMatchSnapshot()
})
