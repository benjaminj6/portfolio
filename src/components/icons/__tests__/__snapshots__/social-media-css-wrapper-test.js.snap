// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Social Media CSS Wrapper snap 1`] = `
preact-render-spy (1 nodes)
-------
<div class="jsx-391102910 jsx-391102910 SocialMediaCssWrapper">
  Test Content
  <JSXStyle
    styleId="391102910"
    css=".jsx-391102910{--fill:var(--local-accent,var(--gray-normal));--size:2rem;}.SocialMediaCssWrapper.jsx-391102910 svg{height:var(--size);width:var(--size);}.SocialMediaCssWrapper.jsx-391102910 path{fill:var(--fill);-webkit-transition:0.175s fill ease-in-out;transition:0.175s fill ease-in-out;}"
  >
  </JSXStyle>
</div>

`;
